content/docs/user-guide/gems/reference/debug/imgui.md

```diff
@@ -1,8 +1,104 @@
+
+## Accessing ImGui menus
+
+ImGui can be accessed by pressing the **Home** key in the editor, game or server launchers.
+
+![ImGui Menus in Viewport](/images/user-guide/gems/reference/debug/imgui_menus.png)
+
+> Server launchers support ImGui if rendering is active. ImGui requires the `rhi` cvar to not be null.
+
+You can use a keyboard, mouse or controller to navigate the ImGui menus.
+
+### Keyboard navigation
+
+For keyboard use the arrow keys to navigate the menus and **Space** to activate or select an option.
+
+### Game controller navigation
+
+You can use the `imgui_EnableController` [CVAR](#cvars) to enable game controller input support, or use the `imgui_EnableControllerMouse` [CVAR](#cvars) to simulate mouse input with a game controller.
+
+The following inputs are supported when using a game controller:
+
+| Input                 | Description                                                                                                       | 
+|-----------------------|-------------------------------------------------------------------------------------------------------------------|
+| D-Pad                 | Move Focus / Navigation <br/>Tweak values (when activated with A) <br/> Resize Window (when holding X)            |
+| Left Stick            | Scroll <br> Move Window (when holding X)                                                                          |
+| X (Left Face Button)  | Tap: Toggle Menu<br>Hold + L1/R1: Focus Window<br>Hold + D-Pad: Resize Window<br />Hold + Left Stick: Move Window |
+| Y (Upper Face Button) | Exit text / on-screen keyboard                                                                                    |
+| B (Right Face Button) | Cancel / Close / Exit                                                                                             |
+| A (Lower Face Button) | Activate / Open / Toggle <br/>Tweak values with D-Pad<br/>(+ L1/R1 to tweak slower/faster)                        |
+
+The following inputs are supported when simulating mouse input with a game controller:
+
+| Input                 | Description               | 
+|-----------------------|---------------------------|
+| Left stick            | Move Mouse Pointer        |
+| A (Lower Face Button) | Left Mouse Button (Btn1)  |
+| B (Right Face Button) | Right Mouse Button (Btn2) |
+
+You can adjust the sensitivity using `imgui_ControllerMouseSensitivity`
+
+## Discrete input mode
+When ImGui is visible, input is sent to both ImGui and the O3DE game/editor. Sometimes, it is more desirable to only control either ImGui or the game/editor at any given time. To facilitate this, the ImGui Gem supports a discrete input mode.
+
+By default, discrete input mode is off. It can be turned on in the ImGui O3DE menu, or with the [CVAR](#cvars) `imgui_DiscreteInputMode`
```

Comment:
```suggestion
By default, discrete input mode is off. It can be turned on in the ImGui O3DE menu, or with the [CVAR](#cvars) `imgui_DiscreteInputMode`.
```